app/main.py

```diff
@@ -1,8 +1,26 @@
+# from __future__ import annotations
+
+
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person in people:
+        Person(person["name"], person["age"])
```

Comment:
Note that this is a list that you must return, so you can use list comprehension and return it at the end instead of using .values()